### PR TITLE
Add Create Inventory endpoint

### PR DESCRIPTION
**Pull request by vkrishna1997**
- Implemented and tested the Create Inventory endpoint
- Throws 201 Created on success and 409 Conflict on existing key pair conflict
- Added tests (success and conflict)
-  Added check_content_type utility to separate utility file

### CODE REVIEW

File: tests/test_routes.py [hunk 1]
```diff
@@ -43,3 +58,44 @@ def test_index(self):
...
+
+    def test_create_inventory_success(self):
+        """It should Create a new Inventory item"""
+        test_inventory = InventoryFactory()
+        logging.debug("Test Inventory create successful: %s", test_inventory.serialize())
+        response = self.client.post(BASE_URL, json=test_inventory.serialize())
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+
+        # Make sure location header is set
+        location = response.headers.get("Location", None)
```
Comment (IrregularHunterX) [inline tests/test_routes.py:70]: What's the location in the header?

**Review by IrregularHunterX — APPROVED** [review]

**Review by MarvinTellis — APPROVED** [review]

File: tests/test_routes.py (hunk 1 above)
Comment (vkrishna1997) [reply tests/test_routes.py]: Create endpoints are supposed to redirect after successful creation but I have to redirect to the List endpoint which doesn't actually exist yet so I'll just make a separate ticket for that in the near future.
Removing it for now! Good catch.

File: tests/test_routes.py [hunk 2]
```diff
@@ -10,30 +10,45 @@
...
 from service import app
-from service.models import db
-from service.common import status  # HTTP Status Codes
+from service.models import Condition, Inventory, db
+from service.common import status
+from tests.factories import InventoryFactory  # HTTP Status Codes
 
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@postgres:5432/postgres"
```
Comment (mmclslwsl) [inline tests/test_routes.py:19]: I think we should use a unified database URI, and add the set up procedure in the README.

Comment (mmclslwsl) [inline tests/test_routes.py:19]: I think we should use a unified database URI, and add the set up procedure in the README.

**Review by mmclslwsl — APPROVED** [review]

**Pull request merged**